packages/flame_yarn/test/parse/tokenize_test.dart

```diff
@@ -2,7 +2,7 @@ import 'package:flame_yarn/src/parse/token.dart';
 import 'package:flame_yarn/src/parse/tokenize.dart';
 import 'package:test/test.dart';
 
-import 'utils.dart';
+import '../utils.dart';
```

Comment:
Does this library not have `flame_lint` yet? :thinking:

Reply:
I think our linter is set up in such a way that it allows relative paths within the test/ directory. See [this test](https://github.com/flame-engine/flame/blob/main/packages/flame/test/rendering/paint_decorator_test.dart) for example.

Reply:
Aah ofc, you can't even do package imports from tests I think?
And the rule only states "for lib imports".
https://dart-lang.github.io/linter/lints/avoid_relative_lib_imports.html